web/crux/prisma/seeds/data/registries.ts

```diff
@@ -27,4 +27,4 @@ export const registries = [
     teamId: constants.TEAM_ID,
     type: 'v2',
   },
-] as Registry[]
+] as Omit<Registry, "id">[]
```

Comment:
If you omit the id, in line 19 the `id: constants.REGISTRY_ID,` will be seeded?

Reply:
Yes, it will be seeded, I have tested the seeder